# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/PBackgroundIDBCursor.ipdl
# Commit: 89817a19f3b3
# Full Hash: 89817a19f3b38187bca7eecd50cbf39867c4dad8
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2019-11-05 22:07:34
# Regressor Bug: 1168606
# File Overlap Count: 1
# Description:
#   Bug 1168606 - Send extra records with every ObjectStoreCursorResponse if enabled by pref. r=ttung,asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D43252
# ==============================================================================

diff -r 2f0447b228c7 -r 89817a19f3b3 dom/indexedDB/PBackgroundIDBCursor.ipdl
--- a/dom/indexedDB/PBackgroundIDBCursor.ipdl	Tue Nov 05 14:40:33 2019 +0000
+++ b/dom/indexedDB/PBackgroundIDBCursor.ipdl	Tue Nov 05 14:40:37 2019 +0000
@@ -73,6 +73,7 @@
   Key objectKey;
 };
 
+// TODO: All cursor responses must be arrays!
 union CursorResponse
 {
   void_t;
